### PR TITLE
feat: navigation,chips,snackbar added

### PR DESCRIPTION
**Pull request by deekshithmd**
Completed navigation, chips and snackbar components

### CODE REVIEW

File: components/snackbar/snackbar.css [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    color: var(--secondary-text-color);
+    border-radius: 5px;
+    padding: 1rem;
+}
+
+.snackbar-button{
+    background-color: transparent;
+    color: var(--snackbar-text-color);
+    border: none;
+    font-size: 1.6rem;
```
Comment (PuneetP16) [inline components/snackbar/snackbar.css:16]: image

It's not readable if you could verify.

**Review by PuneetP16 — APPROVED** [review]
Nice work!!! Liked the navbar. Good to merge.

File: components/snackbar/snackbar.css (hunk 1 above)
Comment (deekshithmd) [reply components/snackbar/snackbar.css:16]: Ok, I will check that, Thank you for the review

File: components/chips/chip.css [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    cursor: pointer;
+    font-size: 2.5rem;
+}
+
+.chip-close:hover{
+    color: var(--close-btn);
+}
+
+.text-chip{
+    padding: 5px 20px;
```
Comment (sanjitsarkar) [inline components/chips/chip.css]: You can use ren or em unit

**Review by sanjitsarkar — APPROVED** [review]

File: components/chips/chip.css (hunk 2 above)
Comment (deekshithmd) [reply components/chips/chip.css]: yes, done. Thank you for the review

**Pull request merged**